packages/shared/src/components/fields/FilterCheckbox.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactElement } from 'react';
+import { Checkbox, CheckboxProps } from './Checkbox';
+
+interface FilterCheckboxProps extends CheckboxProps {
+  description: string;
+}
+
+export function FilterCheckbox({
+  description,
+  children,
+  ...props
+}: FilterCheckboxProps): ReactElement {
+  return (
+    <Checkbox {...props} className="!items-start !typo-callout">
```

Comment:
Maybe we can move these to the default className and merge it?
I think moving forward that's a better practive then overwrites.

So:

```
export const Checkbox = forwardRef(function Checkbox(
  {
    className = 'item-center typo-body font-bold,
    ...props
  }: CheckboxProps,
  ref: LegacyRef<HTMLInputElement>,
): ReactElement {
```

Here we would only add the overwrite classes so they default correctly.
Love to here what you think.

Reply:
It can be a good option but my only concern would be when we just want to add margins and spacing. It will automatically remove the base classes. Wdyt?

Reply:
Hmm yeah good point, not sure how often we override them.
Another approach could be className as object, but it would require more refactor.

Mostly down the line this is pretty bad to keep doing as it will come back to bite us.

Reply:
I agree. Hopefully we can figure out a way to avoid the `important` overrides.